src/containers/hoc/Layout/Layout.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
```

Comment:
This file should not be placed under HoC directory. It is not a higher-order component. We can talk about what it is during our mentoring sessions.